src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -3025,6 +3025,71 @@ def test_aks_create_with_auto_upgrade_channel(self, resource_group, resource_gro
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_create_with_node_os_upgrade_channel(self, resource_group, resource_group_location):
```

Comment:
Queued a [pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=67359954&view=results) to test these newly added test cases.